src/IsoscrollerNode.js

```diff
@@ -95,7 +95,7 @@
       this.textCanvas.height = 128;
 
       this.textCtx = this.textCanvas.getContext('2d');
-      this.textCtx.font = '40pt vcr';
+      this.textCtx.fontFamily = '40pt vcr';
```

Comment:
We talked about this IRL a bit, seems quite weird. Aleks will investigate more and see if he figures something out.